clic/src/cleFlagExistingLabelsKernel.cpp

```diff
@@ -0,0 +1,34 @@
+/*  CLIc - version 0.1 - Copyright 2020 Stéphane Rigaud, Robert Haase,
+*   Institut Pasteur Paris, Max Planck Institute for Molecular Cell Biology and Genetics Dresden
+*
+*   CLIc is part of the clEsperanto project http://clesperanto.net 
+*
+*   This file is subject to the terms and conditions defined in
+*   file 'LICENSE.txt', which is part of this source code package.
+*/
+
+
+#include "cleFlagExistingLabelsKernel.h"
+#include "cleSetKernel.h"
+
+namespace cle
+{
+
+void FlagExistingLabelsKernel::SetInput(Object& x)
+{
+    this->AddObject(&x, "src");
+}
+
+void FlagExistingLabelsKernel::SetOutput(Object& x)
+{
+    this->AddObject(&x, "dst");
+}
+
+void FlagExistingLabelsKernel::Execute()
+{
```

Comment:
Before executing this kernel, we should `set` all pixels to 0. Background is: the kernel does not write all pixels to the vector, it only writes ones. 

See also: https://github.com/clEsperanto/pyclesperanto_prototype/blob/master/pyclesperanto_prototype/_tier2/_flag_existing_intensities.py#L14

Reply:
Not sure if this is relevant in CLIc. `dst` Buffer seems to be created at 0. I should verify this!

I added the `Set` to 0 anyway for security. Maybe removed if not relevant in future.